accounts/ethstore/src/presale.rs

```diff
@@ -31,15 +31,15 @@ pub struct PresaleWallet {
 
 impl From<json::PresaleWallet> for PresaleWallet {
 	fn from(wallet: json::PresaleWallet) -> Self {
-		let mut iv = [0u8; 16];
+		let mut iv = [0_u8; 16];
 		iv.copy_from_slice(&wallet.encseed[..16]);
 
-		let mut ciphertext = vec![];
+		let mut ciphertext = Vec::new();
```

Comment:
This is not a clippy-lint, right?

Reply:
it was not, I can drop this.

Reply:
It is fine for me, I like `Vec::new` better but let's keep it open so the others can see this too.